src/dbt_score/formatters/human_readable_formatter.py

```diff
@@ -44,26 +44,34 @@ def evaluable_evaluated(
         """Callback when an evaluable item has been evaluated."""
         if score.value < self._config.fail_any_item_under:
             self._failed_evaluables.append((evaluable, score))
+        if score.value < self._config.fail_any_item_under or self._config.show in [
```

Comment:
We can save `score.value < self._config.fail_any_item_under` to a param like `evaluable_failed`, to re-use in multiple places

Reply:
done:)

Reply:
Nice, thanks!